openfisca_france/__init__.py

```diff
@@ -1,5 +1,7 @@
 # -*- coding: utf-8 -*-
 
+from openfisca_core import random_stuff
```

Comment:
It's a `[TO REVERT]` commit, to prove we use the right version of core. `random_stuff` only exist in the core branch `test-dev-branch`, so the fact that the test pass demonstrates that the trick works. Sorry, I should have explained it in the PR description.